docs/storage.html

```diff
@@ -0,0 +1,89 @@
+
+        <p>For this to work seemlessly, you'll need:</p>
+        <ul>
+            <li>A default storage class.</li>
+            <li>Dynamic provisioning, or a sufficient number of pre-made PersistentVolumes of appropriate size, in the default storage class.</li>
+        </ul>
+
+        <h1>Local Storage</h1>
+        <p>Local storage doesn't support dynamic provisioning, so you'll need to explicitly create PersistentVolumes.</p>
+
+        <h2>Create Volumes</h2>
+        <p>Local storage uses directories on the host machine.  Any path should work as long as the volumes provisioned below are also updated.  You'll need one directory for each RabbitMQ pod.</p>
+        <code>
+            $ mkdir -p /data/volume-0001
+            $ mkdir -p /data/volume-0002
+            $ mkdir -p /data/volume-0003
+        </code>
+
+        <h2>Provision Volumes</h2>
+        <p>Apply the following configuration to your cluster to provision the necessary volumes.  Again, if you're running more or less than three RabbitMQ instances, adjust the number appropriately.</p>
+        <code>
+            apiVersion: storage.k8s.io/v1
+            kind: StorageClass
+            metadata:
+              name: local-storage
+            provisioner:  kubernetes.io/no-provisioner
+            volumeBindingMode: WaitForFirstConsumer
+            ---
+            apiVersion: v1
+            kind: PersistentVolume
+            metadata:
+              name: pv0001
+            spec:
+              capacity:
+                storage: 1Gi
+              accessModes:
+                - ReadWriteOnce
+              persistentVolumeReclaimPolicy: Retain
+              storageClassName: local-storage
+              hostPath:
+                path: "/data/volume-0001"
+            ---
+            apiVersion: v1
+            kind: PersistentVolume
+            metadata:
+              name: pv0002
+            spec:
+              capacity:
+                storage: 1Gi
+              accessModes:
+                - ReadWriteOnce
+              persistentVolumeReclaimPolicy: Retain
+              storageClassName: local-storage
+              hostPath:
+                path: "/data/volume-0002"
+            ---
+            apiVersion: v1
+            kind: PersistentVolume
+            metadata:
+              name: pv0003
+            spec:
+              capacity:
+                storage: 1Gi
+              accessModes:
+                - ReadWriteOnce
+              persistentVolumeReclaimPolicy: Retain
+              storageClassName: local-storage
+              hostPath:
+                path: "/data/volume-0003"
+            ---
+        </code>
+
+        <h2>Update RabbitMQ Spec</h2>
+        <p>Change the <code>storageClassName</code> in your RabbitMQ instance spec to reference <code>local-storage</code> rather than <code>rook-ceph-block</code>.</p>
```

Comment:
maybe add another short section about rook-ceph-block

Reply:
I added a bit; I'm not sure how much detail you wanted so let me know if there's something in particular I should address.